src/buttonGroup/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import cc from 'classcat'
+
+import style from 'buttonGroup/style'
+
+export interface ButtonGroupProps {
+  readonly children: JSX.Element[]
+  readonly className?: Classcat.Class
+  readonly isInline?: boolean
+  readonly isReverse?: boolean
+}
+
+export default ({
+  children,
+  className = '',
+  isInline = false,
+  isReverse = false,
+}: ButtonGroupProps) => {
+  const classNames = [
+    'kirk-button-group',
+    className,
+    {
+      'kirk-button-group-row': isInline,
+      'kirk-button-group-column': !isInline,
+      'kirk-button-group-reverse': isReverse,
+    },
+  ]
+  return (
+    <div className={cc(classNames)}>
+      {children}
```

Comment:
The ButtonGroup is not outputting buttons? Can you unsure the type of the Children is an array of Buttons?

Reply:
I don't think I can restrict the type of children if they are already instanced (they are all JSX.Element), I'll look into it.